tf2_yolov4/model.py

```diff
@@ -35,6 +35,9 @@ def YOLOv4(
         yolo_score_threshold (float between 0. and 1.): Boxes with score lower than this threshold will be filtered
             out during non max regression.
     """
+    if (input_shape[0] % 32 != 0) | (input_shape[1] % 32 != 0):
+        raise Exception("Input shapes for Height and Width are not multiple of 32")
```

Comment:
Wouldn't it be better to raise an `ValueError`exception instead of the unprecise  `Exception`? See https://stackoverflow.com/questions/256222/which-exception-should-i-raise-on-bad-illegal-argument-combinations-in-python/256260

Reply:
Input shapes -> Input shape

Reply:
In the error message, I would recall the provided input_shape:

```
f"Provided input shape {input_shape} is not a multiple of 32"
```